src/uri/builder.rs

```diff
@@ -78,6 +78,14 @@ impl Builder {
         })
     }
 
+    /// Set the port number for URI, will be part of `Authority`
+    pub fn port(self, port: u16) -> Self {
```

Comment:
This could probably be generic over something that can be converted into `Port`, so a user can easily pass `3000`, or `authority.port()`, etc.

Reply:
I can not see how to be generic over `Port` and `u16`, so I added `port_u16`.

Reply:
Fixed